src/core/messages.ts

```diff
@@ -26,6 +26,26 @@ import EventEmitter from './utils/event-emitter.js';
  */
 interface MessageEventsMap {
   [MessageEvents.Created]: MessageEventPayload;
+  [MessageEvents.Updated]: MessageEventPayload;
+  [MessageEvents.Deleted]: MessageEventPayload;
+}
+
+// This function is used to convert the message action from Ably to the ChatMessageEvents enum
+export function messageActionToMessageEvent(action: Ably.MessageAction): MessageEvents {
+  switch (action) {
+    case 'MESSAGE_CREATE': {
```

Comment:
Can we make these enums, save us repeating ourselves?

Reply:
I have added the following
```ts
/**
* Enum that maps message actions to message events.
  */
enum MessageActionsToEvents {
  'MESSAGE_CREATE' = MessageEvents.Created,
  'MESSAGE_UPDATE' = MessageEvents.Updated,
  'MESSAGE_DELETE' = MessageEvents.Deleted,
  }
```